packages/integration-sdk-runtime/src/storage/FileSystemGraphObjectStore/__tests__/FileSystemGraphObjectStore.test.ts

```diff
@@ -514,7 +514,7 @@ describe('iterateEntities', () => {
     const nonMatchingEntities = times(25, () =>
       createTestEntity({ _type: uuid() }),
     );
-    const matchingEntities = times(25, () =>
+    const matchingEntities = times(25000, () =>
```

Comment:
Is there a reason to have this be so big? 25K entities

Reply:
Nope, I wanted to exercise the concurrency to see if there was any improvement. I can bring it back down